code-scanning/properties/scorecards.properties.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "OSSF Scorecards supply-chain security analysis",
+    "creator": "Open Source Security Foundation (OpenSSF) - https://github.com/ossf",
+    "description": "Scorecards is a static analysis tool to assess the security posture of your project",
+    "iconName": "scorecards",
+    "categories": [
+        "code-quality", "testing",
```

Comment:
For this to show up as a code scanning workflow it needs to have `Code Scanning` as the first category. 🙂 

And for the rest of the categories, the only way these are currently used is to indicate support for specific languages. If you just have `["Code Scanning"]` for the list then it will show up in the setup page for all repositories.

Reply:
I see. I thought some of the additional categories were used for the description if a user clicks. If that's not the case, I'll just add `Code Scanning`. Thanks!

Reply:
Done.